webapp/src/css/enketo/enketo.scss

```diff
@@ -12,6 +12,14 @@
 @import "../../../node_modules/enketo-core/src/sass/core/main";
 @import "../../../node_modules/enketo-core/src/sass/core/pages";
 @import "widgets";
-@import "../../../node_modules/enketo-core/src/sass/core/slider";
 @import "../../../node_modules/enketo-core/src/sass/core/responsive";
 @import "../../../node_modules/enketo-core/src/sass/formhub/main";
+
+form {
+  #form-languages {
+    display: none;
+  }
+  .option-label {
+    white-space: nowrap!important;
```

Comment:
Never use `!important` - find another way to be more specific than the underlying definition.

Reply:
Done.